analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/MethodDescription.java

```diff
@@ -11,6 +11,7 @@ public record MethodDescription(
     MemberDescription member,
 
     @JsonProperty("ReturnType")
+    @JsonInclude(Include.NON_NULL)
```

Comment:
wont non_empty be better here for the off chance the analyzer outputs an empty string?

Reply:
Changed. I wouldn't expect that to happen - I would say the analyzer has a bug in that case - but that means we can use whichever we prefer, so this is now `Include.NON_EMPTY`.